src/content/guides/author-libraries.md

```diff
@@ -305,7 +306,7 @@ W> With webpack 3.5.5, using `libraryTarget: { root:'_' }` doesn't work properly
 
 ### Final Steps
 
-Optimize your output for production by following the steps in the [production guide](/guides/production). Let's also add the path to your generated bundle as the package's `main` field in with our `package.json`
+Optimize your output for production by the following steps in the [production guide](/guides/production). Let's also add the path to your generated bundle as the package's `main` field in with the `package.json`
```

Comment:
i think it was correct. 
- By Following what?
- the steps

Reply:
Same about `our`, this is article's style no?

Reply:
> i think it was correct.
> 
> * By Following what?
> * the steps

I misunderstood may be you are right.

Reply:
> Same about `our`, this is article's style no?

IMO `our` isn't a good choice in formal documentation.

Reply:
@EugeneHlushko @montogeek please check it again